packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/Reflow/DynamicEventline.4.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import LineGraphPanning from '../../../example/Graphs/LineGraph/Reflow/DynamicEventline?dev-site-example';
+import PropsTable from '@cerner/terra-graphs/lib/components/Line/LineGraph?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseriesPanning.js?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Line/panningData-legend.js?dev-site-codeblock';
```

Comment:
Looks like you have used different dataset in example. https://github.com/cerner/terra-graphs/pull/152/files#diff-28ea285c5fcf316959259ab9ace537af38ef63b604c3ffdd5f7e5a1bc6293e36R7